Source/RefinedPower/ModularPower/Buildings/RPMPTurbineBuilding.cpp

```diff
@@ -275,4 +275,15 @@ int ARPMPTurbineBuilding::netFunc_getCurrentRPM()
 float ARPMPTurbineBuilding::netFunc_getSteamDiscardPercent()
 {
     return mSteamDiscardPercent;
-}
\ No newline at end of file
+}
+
+void ARPMTurbineBuilding::netFunc_setSteamDiscardPercent(float value) {
+	auto rco = Cast<URPMPTurbineBuildingRCO>(
+		Cast<AFGPlayerController>(GetWorld()->GetFirstPlayerController())->GetRemoteCallObjectOfClass(
+			URPMPTurbineBuildingRCO::StaticClass()));
+
+	if (rco)
+	{
+		rco->SetSteamDiscard(this, value);
+	}
```

Comment:
Is just `mSteamDiscardPercent = value;` Not enought?

Reply:
maybe. but this is basically a copy of the function that sets the value when the user changes it

Reply:
Yeah thats the reason i am concerned because RCO whatever it does is somehow related to the player and so maybe does not works when done automatically. I have no idea too

Reply:
If it doesnt work then its still an easy fix...

Reply:
Im not familiar with UE and writing code that work with other peoples code without getting a detailed explanation on how to use said code and what it does... but implementing it that way should theoretically work and if it doesnt then just change it so it works.

Reply:
This was no attack to your code i created the same PR until i saw yours. It was more like a question 👍🏻

Reply:
i didnt take it as attack either

Reply:
RCO’s are short for Remote Call Objects it’s a way of the client accessing a UI to send data to the server. RCO calls should be ok but there is better ways of doing it.

Reply:
For example I’m not sure if FIN works from client side when executing the lua code or server side that’s why I got to test

Reply:
It's fine but please tell me the outcome so that  can start Implementing other missing functions for FIN. I'm sure it would take some load off of you Devs, since you have to work on more than one mod